### ISSUE TITLE
MTBE

### ISSUE THREAD

**Issue by JJBarley** [usnistgov/REFPROP-issues#133]
Hi REFPROP team,
Are there any plans to include the thermo-properties for MTBE (2-Methoxy-2-methylpropane) in REFPROP? If not, any suggested references?
Many thanks
Jon

**Comment by ianhbell** [comment]
I don't believe we have any high-accuracy EOS for MTBE.  I think there have been efforts to include MTBE in surrogate modeling of fuels though.  Maybe @marciahuber has some input to share here?

**Comment by marciahuber** [comment]
I do not recall any modeling of MTBE here (but my memory is not 
perfect). Tom Bruno did distillation measurements on several fuels with 
a variety of oxygenates and we developed surrogate models; however  I do 
not think MTBE was one of the oxygenates 
https://dx.doi.org/10.1021/ef2003415,   https://dx.doi.org/10.1021/ef900215v 
https://dx.doi.org/10.1021/ef2003427  For the oxygenates I used a tool in NIST 
ThermoData Engine https://trc.nist.gov/tde.html to develop preliminary 
Helmholtz-form EOS. I think that this approach would work for MTBE as well.
